server/src/platform/drbg/helpers.rs

```diff
@@ -0,0 +1,52 @@
+use super::Drbg;
+
+/// Implement `fill_bytes` via `next_u64` and `next_u32`, little-endian order.
+///
+/// The fastest way to fill a slice is usually to work as long as possible with
+/// integers. That is why this method mostly uses `next_u64`, and only when
+/// there are 4 or less bytes remaining at the end of the slice it uses
+/// `next_u32` once.
+pub fn fill_bytes_via_next<R: Drbg>(
+    rng: &mut R,
+    additional_input: &[u8],
+    dest: &mut [u8],
+) -> Result<(), R::Error> {
+    let mut left = dest;
+    while left.len() >= 8 {
+        let (l, r) = { left }.split_at_mut(8);
+        left = r;
+        let chunk: [u8; 8] = rng.next_u64(additional_input)?.to_le_bytes();
+        l.copy_from_slice(&chunk);
+    }
```

Comment:
Consideration:
Somehow I would love to optimize this - for example if DRBG natively supports only next_u32(), we would use a code variation only having a loop calling u32's. But not sure if there is a good rust way of doing this.